### ISSUE TITLE
Deactivation and Deinstallation not possible 

### ISSUE THREAD

**Issue by mue368** [backdrop-contrib/multifield#55]
After installing the module, I set up this field once in a content type as a test, but never filled it with values. As it did not meet my expectations, I deleted it again in the content type. An Ajax error was displayed, of which I unfortunately do not have a screenshot. But here is the entry in the log:
![Screenshot 2024-01-20 145829](https://github.com/backdrop-contrib/multifield/assets/58108879/d837dacf-16f8-4d3c-b4e8-6d005e2940a0)
However, the entry in the content type has since been deleted. I now want to deactivate this unnecessary module again and then uninstall it. Unfortunately, this is not possible because the module administration displays "pending deletion"
![Screenshot 2024-01-20 144413](https://github.com/backdrop-contrib/multifield/assets/58108879/a79bf66b-a77d-4d6c-bd5d-dc41e15d6595)
However, there is no relevant entry anywhere and the field is not used anywhere:
![Screenshot 2024-01-20 144507](https://github.com/backdrop-contrib/multifield/assets/58108879/7b8b385f-e1b1-4766-a56f-9ccb8d887bfa)
How can I uninstall the module?
Thanks

**Comment by laryn** [comment]
Try running Cron to finish the deletion. See if that gets you past.

**Comment by argiepiano** [comment]
Have you deleted all its subfields before attempting to delete the multifield?

**Comment by mue368** [comment]
Yes, I had checked that all subfields had been deleted in advance and also performed a cron run several times. Of course, all caches have also been deleted several times.

**Comment by laryn** [comment]
@mue368 Did this ever resolve? I may be making a new release in the relatively near future and if this is an ongoing issue I'll try to resolve it before I do so.
